migrations/__tests__/00003_account-data-settings.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { TestDbFactory } from '@/__tests__/db.factory';
+import { PostgresDatabaseMigrator } from '@/datasources/db/postgres-database.migrator';
+import { faker } from '@faker-js/faker';
+import postgres from 'postgres';
+
+interface AccountRow {
+  id: number;
+  group_id: number;
+  created_at: Date;
+  updated_at: Date;
+  address: `0x${string}`;
+}
+
+interface AccountDataTypeRow {
+  id: number;
+  created_at: Date;
+  updated_at: Date;
+  name: string;
+  description: string;
+  is_active: boolean;
```

Comment:
Is this a defined requirement? It's somewhat of a duplication of feature flags and the client equally defines what features they enable via the Config. Service.

Reply:
I am happy with the justification but my main concern was the manual step required to (de-)activate a data type but [you gave clarity to that](https://github.com/safe-global/safe-client-gateway/pull/1715/files#r1668439084). I'm happy to stay with this implementation and we can iterate on it later if necessary.